docs/current/guides/snippets/use-services/expose-service-containers-to-host/index.ts

```diff
@@ -0,0 +1,35 @@
+import { connect, Client } from "@dagger.io/dagger"
+import fetch from "node-fetch"
+
+connect(
+  async (client: Client) => {
+    // create HTTP service container with exposed port 8080
+    const httpSrv = client
+      .container()
+      .from("python")
+      .withDirectory(
+        "/srv",
+        client.directory().withNewFile("index.html", "Hello, world!")
+      )
+      .withWorkdir("/srv")
+      .withExec(["python", "-m", "http.server", "8080"])
+      .withExposedPort(8080)
+      .asService()
+
+    // expose HTTP service to host
+    const tunnel = client
+      .host()
+      .tunnel(httpSrv)
+      .start()
```

Comment:
start() need to be executed async
```suggestion
    const tunnel = await client
      .host()
      .tunnel(httpSrv)
      .start()
```